harness/determined/cli/workspace.py

```diff
@@ -118,7 +118,7 @@ def list_workspace_projects(args: Namespace) -> None:
 
     sort_key = args.sort_by
     sort_order = args.order_by
-    offset = args.offset
+    offset = args.offset or 0  # No passed offset is interpreted as a 0 offset
```

Comment:
should we change the default in `pagination_args` instead?

Reply:
I really don't know. The same `--offset` is used in other methods, too. And even though `list_workspace_projects` thinks `0 == None`, I can't promise that every other method will, too (even though if they didn't, I'd be surprised and disappointed).  Could we call it outside the scope of this PR?